android/app/src/main/java/com/dismoi/scout/Floating/Layout/Base.java

```diff
@@ -0,0 +1,54 @@
+package com.dismoi.scout.Floating.Layout;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.WindowManager;
+import android.widget.FrameLayout;
+
+public class Base extends FrameLayout {
+  private WindowManager windowManager;
+  private WindowManager.LayoutParams params;
+  private Coordinator layoutCoordinator;
+
+  public void setLayoutCoordinator(Coordinator layoutCoordinator) {
+    this.layoutCoordinator = layoutCoordinator;
+  }
+
+  Coordinator getLayoutCoordinator() {
+    return layoutCoordinator;
+  }
+
+  public void setWindowManager(WindowManager windowManager) {
+    this.windowManager = windowManager;
+  }
+
+  public void create(WindowManager getWindowManager, WindowManager.LayoutParams layoutParams, Coordinator layoutCoordinator) {
```

Comment:
This looks like a constructor and the job of a constructor, so it's should probably one.